fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/ConcreteAvailability.java

```diff
@@ -105,4 +105,16 @@ public SimplifiedIntervalList getAvailableIntervals(DataSourceConstraint constra
     protected Map<String, List<Interval>> getLatestAvailableIntervalsByTable() {
         return metadataService.getAvailableIntervalsByTable(name);
     }
+
+    public DataSourceMetadataService getMetadataService() {
+        return metadataService;
+    }
+
+    public TableName getName() {
+        return name;
+    }
+
+    public Set<String> getCachedColumnNames() {
```

Comment:
Probably should make this `protected`, since it's intended as a helper method more than anything else, right?